recipe/meta.yaml

```diff
@@ -31,7 +34,7 @@ test:
 
 about:
   home: https://github.com/harlowja/fasteners
-  license: Apache License Version 2.0
+  license: Apache Version 2.0
```

Comment:
Also could drop "Version".

While we are at it, could we please add `license_file: LICENSE` to package the license from the source code?